packages/perseus/src/util/graphie.test.ts

```diff
@@ -1320,6 +1322,50 @@ describe("Graphie drawing tools", () => {
             expect(onSetDrawingAreaAvailable).toHaveBeenCalledWith(false);
         });
     });
+
+    describe("getMousePx", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [10, 10]],
+            [{left: 20, top: 10}, {pageX: 40, pageY: 40}, [20, 30]],
+        ])(
+            "should return the mouse position in pixel coordinates relative to graph",
+            (graphPosition, mouseEvent, expectedPixelCoord) => {
+                const graphie = createAndInitGraphie();
+                jest.spyOn($.fn, "offset").mockReturnValue(graphPosition);
+
+                const mousePx = graphie.getMousePx(mouseEvent);
+
+                expect(mousePx).toEqual(expectedPixelCoord);
+            },
+        );
+    });
+
+    describe("getMouseCoord", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [2, 8]],
+            [{left: 20, top: 10}, {pageX: 30, pageY: 20}, [2, 8]],
```

Comment:
I just realized... if one of these tests fails, is there any way to tell which one is failing? They have the same description, and the line numbers won't help either...

Reply:
Um, yeah. Good point. Updated!